gpu_support/nvidia/link_nvidia_host_libraries.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+# This script links host libraries related to GPU drivers to a location where
+# they can be found by the EESSI linker
+
+# Initialise our bash functions
+TOPDIR=$(dirname $(realpath $BASH_SOURCE))
+source "$TOPDIR"/../../scripts/utils.sh
+
+# We rely on ldconfig to give us the location of the libraries on the host
+command_name="ldconfig"
+# We cannot use a version of ldconfig that's being shipped under CVMFS
+exclude_prefix="/cvmfs"
+
+found_paths=()
+# Always attempt to use /sbin/ldconfig
+if [ -x "/sbin/$command_name" ]; then
+    found_paths+=("/sbin/$command_name")
+fi
+IFS=':' read -ra path_dirs <<< "$PATH"
+for dir in "${path_dirs[@]}"; do
```

Comment:
As first part of this loop, do:
```
    if [ "$dir" = "/sbin" ]; then
        continue  # we've already checked for $command_name in /sbin, don't need to do it twice
    fi
```
Otherwise, if `/sbin` is also on `PATH`, `/sbin/ldconfig` gets added twice to `found_paths`:
```
Found ldconfig in the following locations:
- /sbin/ldconfig
- /sbin/ldconfig
```

Reply:
Adopted in https://github.com/EESSI/software-layer/pull/368/commits/480d35643c7740c48f5508303e1cccadf04aae76